ansible/roles/ps4-controller/templates/ds4drv.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=ds4drv daemon
+Requires=bluetooth.service
+After=bluetooth.service
+
+[Service]
+EnvironmentFile=/etc/donkeycar/ds4drv.env
+ExecStartPre=/bin/sleep 5
+ExecStart=/home/pi/venv/bin/ds4drv --controller-limit 1 --led ${controller_color} --config /etc/donkeycar/ds4drv.conf --bindings donkeycar
```

Comment:
`controller_color` isn't the name of the environment variable declared in `/etc/donkeycar/ds4drv.env`, it's `$CONTROLLER_LED_COLOR`

Reply:
we should put which variable

Reply:
fixes here [](https://github.com/mdl29/donkeycarLPH/pull/39/commits/5e0010d2e5f179361a56a3500015a19dea60a2d9)[5e0010d](https://github.com/mdl29/donkeycarLPH/pull/39/commits/5e0010d2e5f179361a56a3500015a19dea60a2d9)